dlt/destinations/snowflake/snowflake.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, file_path: str, table_name: str, write_disposition: TWriteDis
             stage_name = client.make_qualified_table_name('%'+table_name)
             with client.begin_transaction():
                 # PUT and copy files in one transaction
-                client.execute_sql(f'PUT file://{file_path} @{stage_name}/"{load_id}" OVERWRITE = TRUE')
+                client.execute_sql(f'PUT file://{file_path} @{stage_name}/"{load_id}" OVERWRITE = TRUE, SOURCE_COMPRESSION = GZIP')
```

Comment:
@rudolfix is there a good way to tell whether the file is gzipped? Is that something we need to worry about?  
Just saw this was added, `AUTO_DETECT` doesn't work here.

Reply:
all the files are compressed by default including jsonl. this is already ON in this branch.
looking at this: https://docs.snowflake.com/en/sql-reference/sql/put#optional-parameters
AUTO_COMPRESS=FALSE and SOURCE_COMPRESSION = AUTO_DETECT
what does it mean it does not work? if it indeed does not then we have a problem... we could detect the compression ourselves but some files will be in the buckets and that makes them harder. why they even check that during staging...